addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1591,7 +1591,9 @@ export function isUnbreakable(node) {
                 node.getAttribute('t-value') ||
                 node.getAttribute('t-out') ||
                 node.getAttribute('t-raw'))) ||
-        node.classList.contains('oe_unbreakable')
+        node.classList.contains('oe_unbreakable') ||
+        (node.tagName === "A" && node.getAttribute('role') === "tab") ||
+        (node.classList.contains('btn') || node.getAttribute('role') === "button")
```

Comment:
Hello @ksbh-odoo `btn` is a bootstrap class it is not a good idea to check it based on a bootstrap class directly. it should be something like `node.nodeName === 'A' && node.classList.contains('btn') `